grafana-formula/grafana/files/containers/grafana.container

```diff
@@ -0,0 +1,13 @@
+[Unit]
+Description=Grafana Container
+
+[Container]
+Label=app=grafana
+ContainerName=grafana
+Image=registry.opensuse.org/devel/bci/sle-15-sp6/containerfile/suse/grafana:9.5.8
+Volume=/etc/grafana:/etc/grafana:ro
```

Comment:
this means the grafana config files must exist on the Host OS will be mounted as read-only on the container? To adapt the configuration users should change the config files on the host OS?

Reply:
Yes, the formula takes care of providing the config files.
And yes, users can modify these using the formula or manually.